library/src/main/java/com/google/android/material/motion/runtime/TargetScope.java

```diff
@@ -103,24 +121,31 @@ private int getDetailedState() {
     return state;
   }
 
+  private void removeNamedPlan(String name, NamedPlanPerformance performer) {
```

Comment:
this helper method is pretty useless now. Inline into `commitRemoveNamedPlan()` please